src/model/operation/convolution.cc

```diff
@@ -347,7 +347,7 @@ Tensor GpuConvBackwardW(const Tensor &dy, const Tensor &x, const Tensor &W, cons
   Tensor dW;
   dW.ResetLike(W);
 
-  dy.device()->Exec([dW, dy, x, W, cch](Context * ctx) {
+  dy.device()->Exec([dW, dy, x, &cch](Context * ctx) {
```

Comment:
better use reference & for all args in [ ]

Reply:
ok